composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "require": {
-        "nexmo/client": "^2.0",
+        "vonage/client": "^2.0",
```

Comment:
Should this change specifically be `^2.2 ` because the Vonage namespace didn't exist earlier?

Reply:
Nope, this works. It's pinning to the latest compatible version no lower than 2.0, and since it can match `vonage/client:2.3.0` with the package name it goes off from there.

`vonage/client:^2.0` and `vonage/client:^2.3` (the Vonage release) are functionally the same in composer.

There's also the catch that `vonage/client` and `vonage/client-core` will almost never be the same, which is where I think part of the confusion comes in. `vonage/client` will generally only ever have major releases to keep with `vonage/client-core`. There's only an intermediary release here to get the package name updated.

Reply:
mmm, I still think it's confusing. Clearly 2.3 does fulfil ^2.0 but I worry that there could be a combination of dependencies that force 2.0. If the package was nexmo in 2.0, and 2.1, but vonage in 2.2 (supporting nexmo also), then even if we've released vonage 2.0 to be the same as vonage 2.2, I think we're really muddying the waters, especially if someone needs a fix that's in 2.1 and it exists in one 2.0 but not the other ..........

Reply:
Upped all the `composer.json` entries to `vonage/client:^2.3`